1_The_Physical/Getting_Started_Emergent_Design/1_Red_Green_Refactor/1_1_Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -2,15 +2,9 @@ export function fizzbuzz(input: number) {
   let isDivisibleBy3 = input % 3 === 0;
   let isDivisibleBy5 = input % 5 === 0;
 
-  if (input > 100 || input < 0) {
-    throw new Error("Number out of range");
-  } else if (isDivisibleBy3 && isDivisibleBy5) {
-    return "FizzBuzz";
-  } else if (isDivisibleBy3) {
-    return "Fizz";
-  } else if (isDivisibleBy5) {
-    return "Buzz";
-  } else if (!isDivisibleBy3 && !isDivisibleBy5) {
-    return input.toString();
-  }
+  if (input > 100 || input < 0) throw new Error("Number out of range");
+  if (isDivisibleBy3 && isDivisibleBy5) return "FizzBuzz";
+  if (isDivisibleBy3) return "Fizz";
+  if (isDivisibleBy5) return "Buzz";
+  if (!isDivisibleBy3 && !isDivisibleBy5) return input.toString();
```

Comment:
in line 9, you can just 
`return input.toString()`
because if the input is divisible by any, the execution wouldn't reach this line.